snooty/rstspec.toml

```diff
@@ -779,13 +779,6 @@ help = "Make a kicker (a subheader that is placed on top of the main header)."
 example = """.. kicker:: ${1: string}"""
 argument_type = "string"
 
-[directive."mongodb:mongo-web-shell"]
-help = """An instance of the web version of a Mongo shell"""
-options.version = "mws_version"
```

Comment:
We should also be able to remove the `mws_version` enum, since I believe only this directive referenced it

Reply:
Great callout